sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobClient.java

```diff
@@ -120,7 +208,13 @@ public BlobClient getVersionClient(String versionId) {
      */
     @Override
     public BlobClient getEncryptionScopeClient(String encryptionScope) {
-        return new BlobClient(client.getEncryptionScopeAsyncClient(encryptionScope));
+        EncryptionScope finalEncryptionScope = null;
+        if (encryptionScope != null) {
+            finalEncryptionScope = new EncryptionScope().setEncryptionScope(encryptionScope);
+        }
+        return new BlobClient(this.client, getHttpPipeline(), getAccountUrl(), getServiceVersion(), getAccountName(),
```

Comment:
Need to update the asyncrhonous client here and in `getCustomerProvidedKeyClient`.